recipes/python/assets/create_oracle_asset.py

```diff
@@ -0,0 +1,247 @@
+    with open(file_name, 'r') as file_handle:
+        return json.load(file_handle)
+
+def get_asset_by_id(uri):
+    oracle_db_get_response = requests.get(base_url + uri, headers=headers, verify=False)
+
+    if oracle_db_get_response.status_code != 200:
+        print("\nAPI returned status code: {}. Response: {}".format(oracle_db_get_response.status_code,
+                                                                   oracle_db_get_response.json))
+        raise SystemExit("\nCound not get the Oracle database by the given ID. Script ended.\n")
+
+    print("\nGet Oracle database API response: ", oracle_db_get_response.json())
+
+def verify_asset_creation(createQueryId):
+    create_status_response = None
+    create_status = "IN_PROGRESS"
+    status_check_count = 0
+
+    while create_status == "IN_PROGRESS":
+        time.sleep(2)
+        print("\nChecking the status of the request...")
+        create_status_query = requests.get(asset_service_url + createQueryId, headers=headers, verify=False)
+
+        create_status_response = create_status_query.json()
+
+        if create_status_query.status_code != 200:
+            print("\nAPI returned status code: {}. Response: {}".format(create_status_response.status_code,
+                                                                       create_status_response))
+            raise SystemExit("\nScript ended.\n")
+
+        create_status = create_status_response['data'][0]['attributes']['status']
+
+        print("\nStatus:", create_status)
+
+        status_check_count += 1
+        if status_check_count >= 10:
+            print("\nRequest to create the asset is still being processed. Exiting status check.")
+            break
+
+    print("\nAPI Response for the Asset create request: ", create_status_response)
+
+    if create_status == 'SUCCESS':
+        return  create_status_response['data'][0]['attributes']['workItemResponses'][0]['links']['self']['href']
+    else:
+        return ''
+
+def perform_login(base_url, username, password, domainName, domainType):
+	url = base_url + "/login"
+	req_body = {'userName': username, 'password': password, 'domainName': domainName, 'domainType': domainType}
+	headers = {'Content-Type': content_type}
+
+	print("\nLogin user '{}'.".format(req_body['userName']))
+
+	resp = requests.post(url, headers=headers, json=req_body, verify=False)
+
+	if resp.status_code != 201:
+		print("\nLogin API failed with status code {} and {}".format(resp.status_code, resp.json()))
+		raise SystemExit("\nLogin failed. Please verify that the input parameters are correct and try again.\n")
+
+	print("Login API returned response status code: {}".format(resp.status_code))
+
+	print("Login successful. Returning the JWT.")
+
+	return resp.json()['token']
+
+if __name__ == '__main__':
+    jwt = perform_login(base_url, username, password, domainName, domainType)
+    headers = {'Content-Type': content_type, 'Authorization': jwt}
+
+    if addCredentials != None:

```

Comment:
I think `is` is the preferred way of comparing `None`.

```suggestion
    if addCredentials is not None:

```

Reply:
Changed all the instances of !=